frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -1316,248 +1299,4 @@ function ReaderBookmark:doesBookmarkMatchTable(item, match_table)
     end
 end
 
-function ReaderBookmark:toggleBookmark(pageno)
-    local pn_or_xp
```

Comment:
I guess you reorganized the functions (which is fine) as this whole block is removed, and its stuff is mixed above, making changes harder to review. Trusting you.